src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4278,6 +4278,10 @@ components:
           items:
             $ref: '#/components/schemas/SnapshotRequestContentsModel'
         readers:
+          description: >
+            Deprecated -- use `policies.readers` instead. Any readers specified here will supplement
+            those set in `policies.readers`.
+          deprecated: true
```

Comment:
By marking this field as deprecated, the user can still specify it, but it isn't shown in the example request body and renders thusly in Swagger's request body schema:
<img width="1415" alt="Screen Shot 2022-10-03 at 12 49 14 PM" src="https://user-images.githubusercontent.com/79769153/193633823-6ddee7ca-1d49-4c46-a9e0-5b65d19ebca8.png">